### ISSUE TITLE
Rich Text editor?

### ISSUE THREAD

**Issue by dje04001** [US-CBP/GTAS-UI#504]
Legacy GTAS has a rich text editor for the text input for notes. We should discuss the footprint and utility of adding this functionality

**Comment by gtasautumn** [comment]
Yes, we don't currently have a component for that but we can add one.
